forestflow/pcross.py

```diff
@@ -3,7 +3,7 @@
 from scipy.interpolate import CubicSpline
 
 
-def P1D_Mpc(P3D_Mpc, z, ln_k_perp, kpars, P3D_mode="pol", **P3D_kwargs):
+def P1D_Mpc(P3D_Mpc, z, ln_k_perp, kpars, P3D_mode="pol", P3D_params={}):
```

Comment:
We have vectorized functions for computing P1D very quickly in other files. Furthermore, it is not clear to my why the P1D function is in the Pcross file. It is fine to leave it here for now, but this is one of the things that we should clean up soon.

Reply:
I would keep it for now, but in the future, I would added the flexibility you need to the vectorized function. We do not want different P1D predictions from ForestFlow that are not consistent